docs-tool/src/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import 'prismjs';
```

Comment:
Can we see if we can get rid of this? Not sure why we need both this and `Prism.highlightAll()`. Shouldn't `highlightAll` only be occurring when the code has been rendered anyways?